config/config.example.yml

```diff
@@ -349,13 +349,17 @@ feed_threads: 1
 ## Enable/Disable the polling job that keeps the decryption
 ## function (for "secured" videos) up to date.
 ##
-## Note: This part of the code generate a small amount of data every minute.
+## Note: This part of the code is currently broken, so changing
+## this setting has no impact.
```

Comment:
Could you please add this as a "Note 2"? That way it will be less diff both now and when the function will be fixed :)